test/core/security/credentials_test.cc

```diff
@@ -1028,7 +1025,7 @@ void validate_sts_token_http_request(const grpc_http_request* request,
       absl::StrFormat("%s?%s", test_sts_endpoint_url, body);
   absl::StatusOr<URI> url = URI::Parse(get_url_equivalent);
   if (!url.ok()) {
-    gpr_log(GPR_ERROR, "%s", url.status().ToString().c_str());
+    LOG(ERROR) << url.status().ToString();
```

Comment:
I think we should look into using Abseil Stringfy with gRPC types if you have not.

Reply:
Missed this. Fixed it.